mod/crepair.php

```diff
@@ -8,10 +8,8 @@
 use Friendica\Core\L10n;
 use Friendica\Core\Protocol;
 use Friendica\Database\DBA;
-use Friendica\Model\Contact;
-use Friendica\Model\Profile;
-
-require_once 'mod/contacts.php';
+use Friendica\Model;
+use Friendica\Module\Contact;
```

Comment:
Although this is correct, please write `use Friendica\Module;` and later `Module\Contact::getTabsHTML` to prevent any immediate confusion.